src/main/java/org/opentripplanner/framework/io/JsonDataListDownloader.java

```diff
@@ -30,7 +30,7 @@ public JsonDataListDownloader(
     @Nonnull Function<JsonNode, T> elementParser,
     @Nonnull Map<String, String> headers
   ) {
-    this(url, jsonParsePath, elementParser, headers, new OtpHttpClient());
+    this(url, jsonParsePath, elementParser, headers, new OtpHttpClientFactory().create(LOG));
```

Comment:
There are a few classes like this which are reused across different updaters. Should we pass in the logger to these also instead of creating the own logger instance for the class or should we pass in http client?

Reply:
We discussed this in the dev meeting and didn't find this to be a high priority thing to improve so we will leave this as is and worry about it later if someone finds this to be problematic.